src/OrchardCore.Modules/OrchardCore.Forms/Drivers/FormContentDisplayDriver.cs

```diff
@@ -13,10 +13,14 @@ public class FormContentDisplayDriver : ContentDisplayDriver
         public override Task<IDisplayResult> DisplayAsync(ContentItem model, BuildDisplayContext context)
         {
             var formItemShape = context.Shape;
-            // If the content item contains FormPart add Form Wrapper.
+            // If the content item contains FormPart add Form Wrapper only in Display type Detail
             var formPart = model.As<FormPart>();
-            if (formPart != null)
+            if (formPart != null && context.DisplayType == "Detail")
             {
+                // Add wrapper for content type
+                formItemShape.Metadata.Wrappers.Add($"Form_Wrapper__{model.ContentType}");
```

Comment:
Looks like this gets hit twice, and is generating two `<form>` tags as a result

Reply:
All wrappers are rendered. here `Form_Wrapper` will generate form Tag and `Form_Wrapper__[content type]` will generate inner wrapper that allows to provide additional customization within that form. No need to define form tag again

Reply:
Try with my example code above

Reply:
ah, but have you tried it without creating the second wrapper template?

Try with the coming soon theme. two form tags...

Reply:
@deanmarcussen  It's not related to this PR, May be you have left over old code for ComingSoon. You  have already fixed that earlier in PR #6346

Reply:
Flow module fresh install.

Before 1 form tag:

![Screenshot 2020-10-04 at 09 42 06](https://user-images.githubusercontent.com/13782679/95011091-2ee1b580-0626-11eb-80ec-21976cac8e8b.png)

After 2 form tags:

![Screenshot 2020-10-04 at 09 43 07](https://user-images.githubusercontent.com/13782679/95011095-35702d00-0626-11eb-9386-58e5acdabf55.png)

Reply:
Wow! It seems wrapper fall back to parent shape

Reply:
fixed

Reply:
Now you got it ;)

Looks good. Why did you remove

`id="form_@Model.ContentItem.ContentItemId"`

I thought that looked useful?

Reply:
I removed id as I feel that its better to use `FormElementPart` but It needs changes in migration so will create separate PR